schema.json

```diff
@@ -147,9 +144,5 @@
         "oneOf": { "$ref": "#/definitions/schemaArray" },
         "not": { "$ref": "#" }
     },
-    "dependencies": {
-        "exclusiveMaximum": [ "maximum" ],
-        "exclusiveMinimum": [ "minimum" ]
-    },
     "default": {}
```

Comment:
And with if/the/else it would look more elegant :)

```json
"dependencies": {
  "exclusiveMaximum": {
    "if": { "properties": { "exclusiveMaximum": { "type": "number" } } },
    "then": { "properties": { "maximum": false } },
    "else": { "properties": { "maximum": true } }
  }
}
```